src/ra_server.erl

```diff
@@ -2307,6 +2318,15 @@ add_reply(_, '$ra_no_reply', _, Effects, Notifys) ->
     {Effects, Notifys};
 add_reply(#{from := From}, Reply, await_consensus, Effects, Notifys) ->
     {[{reply, From, {wrap_reply, Reply}} | Effects], Notifys};
+add_reply(#{from := From}, Reply,
+          {await_consensus, Options}, Effects, Notifys) ->
+    Replier = case Options of
```

Comment:
I switched to a map - if we need to add more options in the future I think it will be a better choice, and it's easier to work with than the list here since these options are mutually exclusive 👍